README.md

```diff
@@ -1 +1,3 @@
-# medical-supply-chain-contracts
\ No newline at end of file
+# medical-supply-chain-contracts
+
+[![Build Status](https://travis-ci.org/EspeoBlockchain/medical-supply-chain-contracts.svg?branch=master)](https://travis-ci.org/EspeoBlockchain/medical-supply-chain-contracts)
```

Comment:
Why do we use Travis this time?

Reply:
It's because I wanted to check it out for a small project first. Especially because of the problems you've been running into with Circle CI ;)